ethereum/p2p/src/test/java/org/hyperledger/besu/ethereum/p2p/EthProtocolHelper.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright contributors to Besu.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.ethereum.p2p;
+
+import org.hyperledger.besu.ethereum.p2p.rlpx.wire.Capability;
+
+public class EthProtocolHelper {
```

Comment:
Do we need this class? EthProtocol.LATEST is exactly the same, isn't it?

Reply:
done

Reply:
yes, I don't want to add a dependency here.